templates/pulp.conf.erb

```diff
@@ -17,15 +17,23 @@ SSLInsecureRenegotiation on
 
 WSGIProcessGroup pulp
 WSGIApplicationGroup pulp
-WSGIDaemonProcess pulp user=apache group=apache processes=1 threads=8 display-name=%{GROUP}
+WSGIDaemonProcess pulp user=apache group=apache processes=3 display-name=%{GROUP}
 
 # DEBUG - uncomment the next 2 lines to enable debugging
 #WSGIRestrictStdin Off
 #WSGIRestrictStdout Off
 
 WSGISocketPrefix run/wsgi
-WSGIScriptAlias /pulp/api /srv/pulp/webservices.wsgi
-WSGIImportScript /srv/pulp/webservices.wsgi process-group=pulp application-group=pulp
+WSGIScriptAlias /pulp/api /usr/share/pulp/wsgi/webservices.wsgi
+WSGIImportScript /usr/share/pulp/wsgi/webservices.wsgi process-group=pulp application-group=pulp
+
+<Directory /usr/share/pulp/wsgi>
+<%- if scope.function_versioncmp([@apache_version, '2.4']) >= 0 -%>
```

Comment:
You will need to set this up -- for example: https://github.com/Katello/puppet-katello/blob/c12447f1c01ba837d5ddbe1aeb908e6d18e6ab57/manifests/config.pp#L4